scripts/qiita

```diff
@@ -224,6 +225,18 @@ def load_reference_db(name, version, seq_fp, tax_fp, tree_fp):
                % reference.id)
 
 
+@db.command()
+@click.argument('fp', required=True,
+                type=click.Path(resolve_path=True, readable=True, exists=True))
+@click.option('--table', required=True, type=click.Choice(SUPPORTED_PARAMS),
+              help='The table to add the parameter set')
+def load_parameters(fp, table):
+    """Loads a parameter set to the database"""
+    param = load_parameters_from_cmd(fp, table)
```

Comment:
The function signature for `load_parameters_from_cmd` indicates a non-optional `name` parameter...